esx_service/vmdk_ops.py

```diff
@@ -1426,7 +1426,7 @@ def disk_detach_int(vmdk_path, vm, device):
         logging.warning("%s\n%s", msg, "".join(traceback.format_tb(ex_traceback)))
         return err(msg)
 
-    setStatusDetached(vmdk_path)
+    setStatusDetached(vmdk_path, key=None, value=None)
     logging.info("Disk detached %s", vmdk_path)
```

Comment:
There is no need to pass key and value here as None foe them is already default.